index.js

```diff
@@ -266,27 +266,34 @@ exports.addTasks = (gulp, libraryName, srcGlob, webpackConfig, dtsGlob) => { //e
     });
 
     gulp.task("start", callback => {
-        const webpackPort = 8888;
-        const host = '0.0.0.0';
-
-        const packageInfo = require(path.join(process.cwd(), 'package.json'));
+        portscanner.findAPortNotInUse(8000, 9000, '127.0.0.1', function(error, port) {
+            console.info('AVAILABLE PORT AT: ' + port); // eslint-disable-line no-console
+            fs.writeFile("PORT", port, function(err) {
```

Comment:
Perhaps you want to use writeFileSync here. Also, single quotes.

Where is the PORT file used?